packages/design-picker/src/components/design-picker-category-filter/index.tsx

```diff
@@ -52,17 +55,34 @@ export function DesignPickerCategoryFilter( {
 				orientation="vertical"
 				className="design-picker-category-filter__sidebar"
 			>
-				{ categories.map( ( { slug, name } ) => (
+				{ categories.map( ( { slug, name, previewUrl } ) => (
 					<MenuItem
 						key={ slug }
 						isTertiary
 						isPressed={ slug === selectedCategory }
 						data-slug={ slug }
 						onClick={ () => onSelect( slug ) }
 						tabIndex={ slug === selectedCategory ? undefined : -1 }
-						className="design-picker-category-filter__menu-item"
+						className={ classnames( 'design-picker-category-filter__menu-item', {
+							'has-thumbnail': previewUrl,
+						} ) }
 					>
-						<span className="design-picker-category-filter__item-name">{ name }</span>{ ' ' }
+						{ previewUrl ? (
+							<span className="design-picker-category-filter__item-thumbnail">
+								<span className="design-picker-category-filter__item-thumbnail-inner">
+									<MShotsImage
```

Comment:
It's a little weird to show the options of the generated design in the category filter because they're the designs instead of the filter IMO

Reply:
Agreed as well!